fastlane/Fastfile

```diff
@@ -74,13 +76,13 @@ lane :update_native_dependencies_to_latest do |options|
 end
 
 desc "Update purchases-ios dependency version"
-lane :update_ios_native_version do |options|
+private_lane :update_ios_native_version do |options|
```

Comment:
I changed these to `private_lane` since I think it's better to keep a cleaner API for fastlane, and having the `update_native_dependencies_to_latest` should be enough I think.